DEVELOPMENT.md

```diff
@@ -0,0 +1,242 @@
+# Viam Micro-RDK Module Template
+
+## (In)stability Notice
+
+> **Warning** The Viam Micro-RDK is currently in beta.
+
+## Overview
+
+This document provides advanced development configuration notes to aid
+in the development of Micro-RDK modules and projects, or for working
+on the Micro-RDK itself.
+
+- If you are interested in just installing and using the Micro-RDK,
+  please see the [Viam Installation Documentation](https://docs.viam.com/installation/microcontrollers/).
+
+- These instructions and notes assume a working Micro-RDK development
+  environment. Please see the [Development Setup Documentation](https://docs.viam.com/installation/viam-micro-server-dev/)
+  before proceeding.
+
+## Micro-RDK Development Tips
+
+### Tree-level `cargo` configuration
+
+The micro-rdk monorepo is only part of the development story: when
+developing for or with Micro-RDK, there will be other directories in
+play (e.g. projects generated from the template, etc.), and
+coordinating the state of these many directories and repositories can
+prove troublesome. A carefully placed `.cargo/config.toml` file can
+inject settings for the entire tree of trees, including the micro-rdk
+monorepo, projects, and modules. Simply create a `.cargo` directory as
+a peer of your micro-rdk monorepo and any project directories, and
+then create a `config.toml` file within it. Settings applied in that
+file will now apply globally to all crates inside the micro-rdk as
+well as to peer project directories. Here is an example layout:
+
+```
+.
+├── .cargo
+│   └── config.toml
+├── micro-rdk
+│   ├── ...
+│   ├── examples
+│   ├── micro-rdk
+│   ├── micro-rdk-ffi
+│   ├── micro-rdk-installer
+│   ├── micro-rdk-macros
+│   ├── micro-rdk-server
+│   └── templates
+└── projects
+    ├── module-under-development
+    ├── another-module
+    └── module-consuming-project
+```
+
+Settings made to the `.cargo/config.toml` file will now apply
```

Comment:
I think this sentence is repeated from above in L. 32?

Reply:
Good catch, removed